source/Esf.Website/gulpfile.js

```diff
@@ -139,8 +139,9 @@ gulp.task('compile:less',
                     './node_modules/bootstrap-less'
                 ]
             }))
-            .pipe(gulpConcatCss('styles.css'))
-            .pipe(gulp.dest(CONTENT_DESTINATION));
+        .pipe(gulpConcatCss('styles.css'))
```

Comment:
we need to update the css bunlding as well. Right now  css doesn't get built under release configuration